pkg/misc/slice/chunk_test.go

```diff
@@ -0,0 +1,75 @@
+package slice
```

Comment:
same here

Reply:
Fixed in 49f0d3ed3e0469c4d9595124b4ee10cd207414d1 also